src/slider.jsx

```diff
@@ -94,7 +93,7 @@ let Slider = React.createClass({
   },
 
   componentWillReceiveProps(nextProps) {
-    if (nextProps.value !== null) {
+    if (!nextProps.value) {
```

Comment:
Sure, and it is kind of problematic for the value field.
But then I was wondering, why did you guys have a check for ```null```? Because to me, if no value is sent in a prop, it will always be ```undefined```.
Would there be a problem in doing :
```js
nextProps.value !== undefined
```
everywhere ?